src/simulation/dynamics/DynOutput/boreAngCalc/_UnitTest/test_bore_ang_calc.py

```diff
@@ -141,15 +141,15 @@ def bore_ang_calc_func(testFixture, show_plots, boresightLoc, eulerLoc):
 
     # Execute simulation
     TotalSim.InitializeSimulation()
-    TotalSim.AddVariableForLogging(BACObject.ModelTag + ".boreVecPoint", 1, 0, 2, "double")
+    TotalSim.AddVariableForLogging(BACObject.ModelTag + ".boreVec_Po", 1, 0, 2, "double")
```

Comment:
I would squash all the unit test commits into one.

Reply:
Done